test/spec/components/TemplatingEditor.spec.js

```diff
@@ -14,11 +14,11 @@ import {
 
 import { useEffect, useRef } from 'preact/hooks';
 
-import FeelTemplatingEditor from 'src/components/entries/templating/FeelTemplatingEditor';
+import FeelTemplating from 'src/components/entries/templating/FeelTemplating';
```

Comment:
```suggestion
import FeelTemplating from 'src/components/entries/templating/Templating';
```

forgot to rename it properly